charts/osm/Chart.yaml

```
@@ -14,11 +14,11 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 1.2.0
+version: 1.2.0-rc.1
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
-appVersion: latest-main
+appVersion: 1.2.0-rc.1
```

Only for the appVersion we've been adding a leading `v`, but I'm not sure if it really matters:
```suggestion
appVersion: v1.2.0-rc.1
```